code/quarkus-telemetry/quarkus-telemetry-legume/src/main/resources/application.properties

```diff
@@ -8,14 +8,15 @@ quarkus.hibernate-orm.database.generation=drop-and-create
 # Service
 quarkus.application.name=legume
 
+
 #OTel
-quarkus.otel.tracer.exporter.otlp.endpoint=http://localhost:4317
+quarkus.opentelemetry.tracer.exporter.otlp.enabled=false
```

Comment:
See https://github.com/quarkiverse/quarkus-opentelemetry-exporter/blob/main/docs/modules/ROOT/pages/quarkus-opentelemetry-exporter-azure.adoc What is the way to disable OTLP export now?

Reply:
@brunobat This one has to be used?
https://quarkus.io/guides/opentelemetry
![image](https://github.com/Azure-Samples/java-native-telemetry/assets/14811066/d6738db4-0846-4714-a10c-2297795443ab)

Reply:
But, I am not sure that this configuration is needed because:
1) I have not observed any OTLP error message without this configuration in the Azure tests of https://github.com/quarkiverse/quarkus-opentelemetry-exporter
2) It seems that if no OTLP endpoint is defined then the OTLP exporter is not enabled: https://github.com/quarkusio/quarkus/blob/7eb8479ed7bc097754600590ed7b66aa1640d1a4/extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/exporter/otlp/OTelExporterRecorder.java#L54